packages/ui/components/form-core/src/validate/Validator.js

```diff
@@ -27,12 +28,20 @@ export class Validator extends EventTarget {
     this.__param = param;
     /** @type {ValidatorConfig} */
     this.__config = config || {};
-    /** @type {ValidationType} */
-    this.type = config?.type || 'error'; // Default type supported by ValidateMixin
+    /**
+     * Default type supported by ValidateMixin
+     * @type {ValidationType}
+     */
+    this.type = config?.type || 'error';
+    /**
+     * Way of interacting with the form-field: "enter" | "select" | "upload"
+     * @type {InputMode}
+     */
+    this.inputMode = config?.inputMode || 'enter';
```

Comment:
Can we maybe rename this to interactionMode? Since inputMode is already occuppied: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode

Reply:
InteractionMode is also already occupied in our combobox.

Reply:
Currently went for `inputInteractionMode`

Reply:
`operationMode` maybe?